### PR TITLE
DOC-412 How to add a new notary after the network map is running & ho…

### PR DESCRIPTION
**Pull request by erinking-r3**
…w to sign networkmap

### CODE REVIEW

File: content/en/docs/corda-os/4.6/network-map.md [hunk 1]
```diff
@@ -186,23 +186,23 @@ connectivity is required for zone members, required cryptographic algorithms and
...
-Updating of the parameters by the zone operator is done in two phases:
-1. Advertise the proposed network parameter update to the entire network.
-2. Switching the network onto the new parameters - also known as a *flag day*.
+The zone operator updates the parameters by:
+1. Advertising the proposed network parameter update to the entire network.
+2. Switching the network onto the new parameters. This is known as a *flag day*.
 
-The proposed parameter update will include, along with the new parameters, a human-readable description of the changes as well as the
-deadline for accepting the update. The acceptance deadline marks the date and time that the zone operator intends to switch the entire
+The proposed parameter update includes, a human-readable description of the changes and the
```
Comment (taylorharvey-r3) [inline content/en/docs/corda-os/4.6/network-map.md:197]:
```suggestion
The proposed parameter update includes a human-readable description of the changes and the
```

**Review by taylorharvey-r3 — COMMENTED** [review]
One tiny suggestion but everything else LGTM.

File: content/en/docs/corda-os/4.6/network-map.md (hunk 1 above)
Comment (erinking-r3) [reply content/en/docs/corda-os/4.6/network-map.md:197]: eeek thanks!

**Review by taylorharvey-r3 — APPROVED** [review]
LGTM

**Pull request merged**